src/transforms/stdnames.jl

```diff
@@ -0,0 +1,45 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    StdNames(:spec)
+
+StdNames transform standardizes the column names.
+"""
+struct StdNames <: Stateless
+  spec::Symbol
+end
+
+StdNames() = StdNames(:upper)
+
+isrevertible(::Type{StdNames}) = true
+
+function apply(transform::StdNames, table)  
+  cols = Tables.columns(table)
+  oldnames = Tables.columnnames(cols)
+  spec = transform.spec
+
+  (spec == :camel) && (newstringnames = _camel.(string.(oldnames)))
+  (spec == :snake) && (newstringnames = _snake.(string.(oldnames)))
+  (spec == :upper) && (newstringnames = _upper.(string.(oldnames)))
+
+  newnames = Symbol.(newstringnames)
```

Comment:
Rename struct is defined as:
```
struct Rename <: Stateless
  names::Dict{Symbol,Symbol}
end
```
And its constructors:
```
Rename(names::Pair) = _pairsyms(names) |> Dict |> Rename
Rename(names...) = _pairsyms.(names) |> Dict |> Rename

_pairsyms(x::Pair) = Symbol(first(x)) => Symbol(last(x))
```
We can pass `string => Symbol`, `Symbol => string`, and `string => string` pairs to `Rename`, and it will work. But if we pass them as a dictionary (which is what we are doing in this case), it will throw `MethodError: Cannot convert an object of type String to an object of type Symbol`.

I think this is inconsistent behavior and should be changed. We can update the `Rename` transform to accept strings as part of a dictionary, too. But that is beyond the scope of the PR. @juliohm

Reply:
Yes, we can work on that on a separate PR.